internal/applier/helm.go

```diff
@@ -171,6 +164,10 @@ func (h *Helm) getReleaseState(cl helmclient.ActionInterface, ext *ocv1.ClusterE
 		}
 		return nil, desiredRelease, StateNeedsInstall, nil
 	}
+	if err != nil {
+		return nil, nil, StateError, err
```

Comment:
👍 I see here the return all fine,
I just think would be great that ensure that is covered via tests

Reply:
just added unit tests

Reply:
we have tests now. the new tests for the logic flow in this file seem like pretty good coverage now. thanks!